apps/devhub/tasks.py

```diff
@@ -180,13 +181,24 @@ def handle_upload_validation_result(results, upload_pk, annotate=True):
     now = datetime.datetime.now()
     now_ts = utc_millesecs_from_epoch(now)
     delta = now_ts - upload_start
+    statsd.timing('devhub.validation_results_processed', delta)
+
+    # Scale the upload / processing time by MB size of the package
+    # so we can normalize large XPIs which naturally take longer to validate.
+    size = storage.size(upload.path)
+    scaled_delta = None
+    if size > 0:
+        size_in_mb = Decimal(size) / Decimal(1024 * 1024)
+        scaled_delta = Decimal(delta) / size_in_mb
+        statsd.timing('devhub.validation_results_processed_per_mb',
+                      scaled_delta)
```

Comment:
Do we expect this to be proportional to file size? It doesn't seem to me that a 10MB package should take 10 times longer to validate than a 1MB package. I'm not seeing what benefit this metric gives, if the value is still big then I don't know what that tells us.

When I saw the title of this I was thinking buckets not scale. So validation time <1MB, <5MB, >=5MB for example.